test/behavioral/tests/tabs.ts

```diff
@@ -1,13 +1,15 @@
-import {Snappit, WebDriver, Key} from "snappit-visual-regression";
+import {WebDriver, By, Key} from "snappit-visual-regression";
 import {test, TestContext} from "ava";
 import * as util from "../../common/util";
 
 const getTab = async (
     driver: WebDriver,
-    tabName: string
+    tabID: string
 ) => {
-    const tab = await util.$x(driver, util.selectors.tab, tabName);
-    return tab;
+    await driver.findElement(By.css("hx-tablist")).click();
```

Comment:
The `<hx-tablist>` element doesn't do anything on click.
What purpose does this serve?

Reply:
Removed.